src/buckets/components/createBucketForm/MeasurementSchemaSection.tsx

```diff
@@ -218,14 +250,61 @@ const AddingPanel: FC<AddingProps> = ({
   )
 }
 
-const EditingPanel: FC<PanelProps> = ({measurementSchema, index}) => {
+const EditingPanel: FC<PanelProps> = ({
+  index,
+  measurementSchema,
+  onAddUpdate,
+  toggleUpdate,
+}) => {
+  const [fileErrorMessage, setFileErrorMessage] = useState(null)
+  const [fileError, setFileError] = useState(false)
```

Comment:
names changed for clarity as requested.

Reply:
`fileError` and `setFileError` have not changed yet and I agree that they should be changed to read more like a boolean.

Reply:
Also, nit: `useState` can be called with a type like this `useState<boolean>(false)`

In case of `null` as the initial value, the extra type declaration makes the intention clear on what the type should be, like this: `useState<string>(null)`

Reply:
...changed already, using <string> for the null one